<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>381</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background-color:#34495e;

}
</string>
  </property>
  <widget class="QPushButton" name="start">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>200</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color:none;
border-style:solid;
border-width: 2px;
border-radius: 10px;
border-color: #2ecc71;
text-color:white;
color:white;
}
QPushButton:hover{
background-color:#2ecc71;
}</string>
   </property>
   <property name="text">
    <string>Старт</string>
   </property>
  </widget>
  <widget class="QLabel" name="name">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>-10</y>
     <width>411</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Сколько слов вы желаете повторить?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="question">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
background-color:none;
border-style:solid;
border-width: 3px;
border-radius: 10px;
border-color: #3498db;
color:black;
}
QLineEdit:hover{
border-color:#2ecc71;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>280</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color:none;
border-style:solid;
border-width: 2px;
border-radius: 10px;
border-color: #2ecc71;
text-color:white;
color:white;
}
QPushButton:hover{
background-color:#2ecc71;
}</string>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox{
background-color:none;
}</string>
   </property>
   <item>
    <property name="text">
     <string>Свой словарь</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Словарь программы</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
